core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/iterable/ConsumableBlockingQueue.java

```diff
@@ -24,6 +24,7 @@
 
 import java.util.Iterator;
 import java.util.NoSuchElementException;
+import java.util.Objects;
```

Comment:
Could be static? Also seems like using this doesn't provide any benefit, I'm not sure we should be adding it if the code is otherwise ok, [this isn't it's intended use case](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#isNull-java.lang.Object-).

Reply:
I personally find it easier to read than `null == variable` or `null != variable`. I find the ordering of the conditionals stating with `null` difficult to read as well.

Reply:
I also prefer `null == variable` but it's just a preference thing and there is no right answer. However, the precedent in Gaffer is to use `null == variable` so perhaps we shouldn't have an endless back and forth and edit pre-existing uses, but instead just write new uses how we prefer. Will resolve this conversation though as it doesn't matter too much.